app/components/button-with-confirmation/template.hbs

```diff
@@ -1,2 +1,9 @@
-{{input classNameBindings=":form-control isConfirmed:confirmed:unconfirmed" value=enteredValue name="virtual-domain"}}
-<button {{bind-attr disabled=isUnconfirmed}} class="btn btn-danger" {{action 'submit'}}>{{yield}}</button>
+<div class="row">
+  <div class="col-xs-7">
+    {{input classNameBindings=":form-control isConfirmed:confirmed:unconfirmed" value=enteredValue name="virtual-domain"}}
```

Comment:
This can be just `classBinding`. I believe in ember 1.10 stable or 1.11 this changes and we can start using just `class=...`